### DISCUSSION TITLE
update only payload of specific point

### DISCUSSION THREAD

**Discussion by hwk06023** [qdrant/qdrant-client#610] [Q&A] [answered]
Hi 👋

Does Qdrant support the ability to specify a point as an id of the point and then modify only the payload of that point? 

In my case, I would like to add a new key and value or modify the value of the existing key while keeping the other elements of payload.

I think it's done through upsert, but is it only possible to search and input all existing elements such as vector?

Thanks.

**Comment by hwk06023** [comment]
``` python
qdrant_client.overwrite_payload(
    collection_name="asan",
    # in my case, points=["9fad...(id)"]
    points=[Union[List[Union[int, str, PointId]], Filter, PointsSelector]],
    payload = {other elements of payload, new_key: new_value, ..},
)
```

This method is valid if choose to import and insert information from other existing payloads.
It is good that selecting a target point is strong through Filter.

Maybe it's looking for convenience, but it's not just for "=" and it would be nice to have a more flexible ability to modify payloads in the code, such as "append(extend)".

**Comment by joein** [comment, marked as answer]
Hi @hwk06023 

`overwrite_payload` as in the name, overwrites payload completely
there is another method, with which you can modify payload, it is called `set_payload` [link](https://qdrant.tech/documentation/concepts/payload/?q=set_payload#set-payload)

I'll move the issue to the discussions
